.gitignore

```diff
@@ -1,4 +1,5 @@
 */target
+target/
```

Comment:
is L1 still needed? Did it ever do anything?

Reply:
I believe that was due to the DataAccess folder and can probably be removed

Reply:
That will still be needed. I added line 2 because it also was adding a target directory under the base folder that wasn't getting ignored. the */target will match the dfsclient/target or wsclient/target directories that get generated after running the mvn build.